frontend/lexer_def.py

```diff
@@ -303,6 +303,17 @@ def IsKeyword(name):
   R(r'[0-9]*<>', Id.Redir_LessGreat),
   R(r'[0-9]*>\|', Id.Redir_Clobber),
 
+  R(r'\{[_a-zA-Z][_a-zA-Z0-9]*\}<', Id.Redir_Less),
```

Comment:
I think this is identical to  `VAR_NAME_RE` right?  If so let's use it, maybe with an intermediate like:

```
FD_VAR_NAME = '\{' + VAR_NAME_RE + '\}'
```

Reply:
Thank you! I defined `FD_VAR_NAME` cc491ffb.